inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkUtils.java

```diff
@@ -106,6 +105,16 @@ public static String findFiles(String baseDirName,String pattern) {
         return null;
     }
 
+    /**
+     * get value
+     * @param key
+     * @param defaultValue
+     * @return
+     */
+    public static String getValue(String key, String defaultValue) {
```

Comment:
set default `""` is redundancy. Because `FlinkService` will use value of config file to fill value by judging `endpoint` is empty.

Reply:
done